src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -210,7 +215,7 @@ public function getFolderFiles($basename = false)
     public function getFiles($basename = false, $folder = '')
     {
         $pattern = function_exists('config') ? config('logviewer.pattern', '*.log') : '*.log';
-        $files = glob(storage_path() . '/logs/' . $folder . '/' . $pattern, preg_match('/\{.*?\,.*?\}/i', $pattern) ? GLOB_BRACE : 0);
+        $files = glob(storage_path() . '/logs/' . $pattern, preg_match(self::$patterns['get_files'], $pattern) ? GLOB_BRACE : 0);
```

Comment:
Maybe this pattern may not be separated and should leave here. Not sure though... Do as you want!

Reply:
Well, this files pattern(files) can be used in another function in the future, my suggestion is to move it and in this way can be used for future functions.

Reply:
OK!